app.js

```diff
@@ -3,18 +3,28 @@ const productRouter = require('./routes/productRouter.js');
 const orderRouter = require('./routes/orderRouter.js');
 const pool = require('./models/pool.js');
 const options = require('./optionsDB.js');
+const morgan = require('morgan'); // console.log query info
+const authMiddleWare = require('./common/middleware/authMiddleWare.js');
+const authDTO = require('./common/middleware/authDTO.js');
+const orderDTO = require('./common/middleware/orderDTO.js');
+const validator = require('express-joi-validation').createValidator({});
+const errorHandler = require('./common/middleware/errorHandler.js');
 
 const app = express();
 app.use(express.json());
 app.use(express.urlencoded({extended:false}));
 const PORT = 3000;
 
+app.use(morgan('dev'));
+
 app.use('/products', productRouter);
-app.use('/order', orderRouter);
+app.use('/order', validator.headers(authDTO), authMiddleWare, validator.body(orderDTO), orderRouter);
```

Comment:
Better provide middlewares in the `orderRouter`. Also, please, specify request type inside `orderRouter` and `productRouter` files. Right now you're using `.use()` everywhere. I guess, there should be at least `.get()`.

Reply:
sure. perfect is post here, as it is POST request